README.md

```diff
@@ -34,11 +35,7 @@ cargo run
 ### Tests
 
 ```
-scripts/test.sh
-```
-or 
-```
-cargo test --features=test-utilities
+cargo test
```

Comment:
Is this correct or is a rebase missing?

Reply:
`test_utilities` is not hidden behind a feature flag currently, so it's not necessary to declare it